rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormExecReportDataProvider.groovy

```diff
@@ -255,9 +264,12 @@ class GormExecReportDataProvider implements ExecReportDataProvider, DBExecReport
 
                 if (query.execProjects && seId) {
                     or {
-                        if(execUuids && execUuids.size() > 0){
+                        if(execsIds && execsIds.size() > 0){
                             and{
-                                'in'('executionUuid', execUuids)
+                                List execsIdPartitioned = Lists.partition(execsIds, 10000)
+                                for(def execIdPartition : execsIdPartitioned){
+                                    'in'('executionId', execIdPartition)
+                                }
```

Comment:
Do you need to wrap the 'in' sections in an `or{ }`? otherwise won't it will basically be like

```
and{
 'in'('executionId',[...])
 'in'('executionId',[...])
 'in'('executionId',[...])
}
```
which would not work